spec/rspec/core/pending_example_spec.rb

```diff
@@ -44,7 +44,7 @@
 
       group.run
       example = group.examples.first
-      expect(example.metadata[:pending]).to be_truthy
+      expect(example.metadata[:pending]).to be(true)
```

Comment:
Huh, I would have expected this to be truthy due to the message but must be wrapped somewhere I've forgotten about :joy: